ccswbs/hjckrrh
# Issue 197 - change page title to match pane

Fixes #197

Pulls page title from events view pane (E1), overrides E1 title to be blank, and overrides title for /events page to "Upcoming Events"

## Test Procedure

1. Check that generic "Upcoming Events" page title is showing at sitename/events.
2. Filter events by an existing term --> sitename/events/[term ID]. 
3. Check to see that the page title is now "Events related to [term name]"

![screenshot-localhost-2017-04-13-11-34-56](https://cloud.githubusercontent.com/assets/25161107/25012066/57c015d2-203d-11e7-9d8e-cea4bd9fa92e.png)
